python/ray/experimental/channel/shared_memory_channel.py

```diff
@@ -451,3 +458,110 @@ def close(self) -> None:
         if self.is_local_node(self._reader_node_id):
             self.ensure_registered_as_reader()
         self._worker.core_worker.experimental_channel_set_error(self._reader_ref)
+
+
+@PublicAPI(stability="alpha")
+class MultiChannel(ChannelInterface):
```

Comment:
The name makes sense, but wondering it can be improved further. How does `CompositeChannel` sound?

Reply:
Updated https://github.com/ray-project/ray/pull/45591/commits/f49593c0e3fe262464f8307e5d29cfe0d63b85b1